zerver/tests/test_subs.py

```diff
@@ -640,6 +640,44 @@ def test_acting_user_is_creator(self) -> None:
         created_stream = new_streams[0]
         self.assertEqual(created_stream.creator_id, hamlet.id)
 
+    def test_channel_create_message_exists_for_all_policy_types(self) -> None:
+        """
```

Comment:
Shouldn't the test also include a scenario where the organization and personal preference settings are set to different languages?

Reply:
As suggested on [CZO](https://chat.zulip.org/#narrow/stream/49-development-help/topic/Testing.20translation.20language.20preference.20override.20.2331009), I have manually tested that this message uses org language.

I did `./manage.py makemessages --all` and then after providing translation in the corresponding `*.po` file, I ran `./manage.py compilemessages`.

Reply:
Okay then lgtm.